ffi/java-idsdk/src/main/java/com/iotics/sdk/identity/SimpleConfig.java

```diff
@@ -0,0 +1,92 @@
+package com.iotics.sdk.identity;
+
+import com.google.common.base.Strings;
+import com.google.common.io.Files;
+import com.google.gson.Gson;
+
+import java.io.FileNotFoundException;
+import java.io.Reader;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class SimpleConfig {
+
+    private String seed;
+    private String keyName;
+
+    public static SimpleConfig fromEnv(String prefix) {
+        return new SimpleConfig(System.getenv(prefix + "SEED"), System.getenv(prefix + "KEYNAME"));
```

Comment:
should this call `isValid`? It doesn't seem right that the end user should have to do this themselves.